app/services/user_defined_fields/converter.rb

```diff
@@ -29,6 +29,16 @@ def convert_array(value)
         end
       end
 
+      def convert_bool(value)
+        if value.is_a? String
+          value.to_bool
+        elsif value.in? [true, false]
+          value
+        else
+          nil
```

Comment:
We can probably have this return `false`. [Three-state booleans](https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem) are no fun to deal with.

Reply:
Good call. In that case, I'll also ensure `value.present?` doesn't coerce a `false` into `nil`.